manifest-rhel-9.6.yaml

```diff
@@ -0,0 +1,88 @@
+# Manifest for RHCOS based on RHEL 9.6
+
+rojig:
+  license: MIT
+  name: rhcos
+  summary: RHEL CoreOS 9.6
+
+variables:
+  osversion: "rhel-9.6"
+
+# Include manifests common to all RHEL and CentOS Stream versions
+include:
+  - common.yaml
+
+# CentOS Stream 9 repos for now
+repos:
+  - c9s-baseos
+  - c9s-appstream
+
+# Eventually we should try to build these images as part of the RHEL composes.
+# In that case, the versioning should instead be exactly the same as the pungi
+# compose ID.
+automatic-version-prefix: "9.6.<date:%Y%m%d%H%M>"
+# This ensures we're semver-compatible which OpenShift wants
+automatic-version-suffix: "-"
+
+mutate-os-release: "9.6"
+
+packages:
+ # We include the generic centos release package and fake the red hat os-release
+ # info in a post-process script
+ - centos-release
+
+# Fake out RHEL version in the os-release while waiting for RHEL-9.6 release.
+postprocess:
+  - |
+     #!/usr/bin/env bash
+     set -xeo pipefail
+
+     . /etc/os-release
+     cat > /usr/lib/os-release <<EOF
+     NAME="Red Hat Enterprise Linux"
+     VERSION="9.6 (Plow)"
+     ID="rhel"
+     ID_LIKE="fedora"
+     VERSION="${OSTREE_VERSION}"
+     VARIANT="CoreOS"
+     VARIANT_ID=coreos
+     VERSION_ID="9.6"
+     PLATFORM_ID="platform:el9"
+     PRETTY_NAME="${NAME} CoreOS ${OSTREE_VERSION}"
+     ANSI_COLOR="0;31"
+     LOGO="fedora-logo-icon"
+     CPE_NAME="cpe:/o:redhat:enterprise_linux:9::coreos"
+     HOME_URL="https://www.redhat.com/"
+     DOCUMENTATION_URL="https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9"
+     BUG_REPORT_URL="https://issues.redhat.com/"
+
+     REDHAT_BUGZILLA_PRODUCT="Red Hat Enterprise Linux 9"
+     REDHAT_BUGZILLA_PRODUCT_VERSION=9.6
+     REDHAT_SUPPORT_PRODUCT="Red Hat Enterprise Linux"
+     REDHAT_SUPPORT_PRODUCT_VERSION="9.6"
+     EOF
+     )
+
+
+      rm -f /etc/system-release /etc/os-release /etc/redhat-release
+      ln -s /usr/lib/os-release /etc/os-release
+      ln -s /usr/lib/os-release /etc/system-release
+      ln -s /usr/lib/os-release /etc/redhat-release
+  - |
+      #!/usr/bin/env bash
+      set -xeuo pipefail
+      cat > /tmp/scos-workarounds.cil << EOF
+      ; https://issues.redhat.com/browse/RHEL-49735
+      (typeattributeset cil_gen_require afterburn_t)
+      (typepermissive afterburn_t)
+
+      ; https://issues.redhat.com/browse/RHEL-38614
+      (typeattributeset cil_gen_require coreos_installer_t)
+      (typepermissive coreos_installer_t)
+
+      ; https://issues.redhat.com/browse/RHEL-47033
+      (typeattributeset cil_gen_require systemd_network_generator_t)
+      (typepermissive systemd_network_generator_t)
+      EOF
+      /usr/sbin/semodule -i /tmp/scos-workarounds.cil
+      rm /tmp/scos-workarounds.cil
```

Comment:
Let's see if we can merge https://github.com/openshift/os/pull/1568 so we can drop this.